index.html

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New-Year-party</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  </head>
+  <body>
+    <section class="main-container">
+      <section class="hello-container">
+        <div class="upper">
+          <a href="www.facebook.com" class="facebook"
+            ><img
+              src="./assets/images/facebook-1.svg"
+              alt="facebook icon"
+              class="icon1"
+          /></a>
+          <a href="www.twitter.com" class="twitter"
+            ><img
+              src="./assets/images/twitter-1.svg"
+              alt="twitter icon"
+              class="icon1"
+          /></a>
+          <a href="#" class="upper-link">English</a>
+          <a href="#" class="upper-link">My page</a>
+          <a href="#" class="upper-link">Log out</a>
+        </div>
+        <header class="hello-header">
+          <img
+            src="./assets/images/menu_black_48dp.svg"
+            alt="hamburger menu icon"
+            class="menu-image"
+          />
+          <a href="#sub-heading1" id="logo-header"
+            ><img
+              src="./assets/images/pyramid.svg"
+              alt="party logo"
+              class="party-logo1"
+          /></a>
+          <div class="menu-appear">
+            <p class="close-menu">X</p>
+            <ul class="navigation">
+              <li><a href="#sub-heading1" class="menu-li">Hello</a></li>
+              <li>
+                <a href="#the-program" class="menu-li">The main program</a>
+              </li>
+              <li><a href="#stars" class="menu-li">Our Stars</a></li>
+              <li><a href="#about-section" class="menu-li">About</a></li>
```

Comment:
- [ ] When I click on `About` link, a new page should open with a `footer` and its own `navbar`, but instead I am taken to the different section of the same page with `#about-section` id. I noticed that in project you have only 1 html index file. Please kindly add `about.html` file as well and make sure that the About link takes to `about.html` page. 
- [ ] While writing `about.html` please try to stick  to provided [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).